module/Finna/src/Finna/View/Helper/Root/RecordDataFormatterFactory.php

```diff
@@ -1094,6 +1094,17 @@ protected function getDefaultCoreFields()
                 'context' => ['class' => 'extendedFindingAids']
             ]
         );
+        $setTemplateLine(
```

Comment:
Voisiko vain korvata normaalin Finding Aid:n RecordDataFormatter:ssa? API:ssa täytyy tietty säilyttää vanha.

Reply:
Meinaatko siis, että käyttäisi RecordDataFormatterissa vain getFindingAidsExtended funktiota, ja lisäis sellasen funktion sitten muillekkin formaateille, missä finding aids on mukana?

Reply:
Juuri näin.

Reply:
Ilmeisesti tästä olisi tullut turhan paljon muutoksia, vai?

Reply:
joo sanoisin näin